### DISCUSSION TITLE
Problem with checkout develop

### DISCUSSION THREAD

**Discussion by venteo22** [TheAlbertDev/MASB#76] [Q&A] [answered]
Hello community :)
I don't know why I can't checkout to the branch develop that my mate has created. 
This is the error that I receive. 

<img width="563" alt="Captura de pantalla 2022-03-19 a las 14 34 47" src="https://user-images.githubusercontent.com/79203895/159123246-eaa623df-1624-4723-9fc9-2dc028a829c9.png">

**Comment by TheAlbertDev** [comment, marked as answer]
The problem was solved during a videoconference. The problem was that a branch called `develop/<something>` had been created and that's why it didn't allow to create a `develop` branch. Internally, git stores branches as directories and files. If we do a branch restructure like:

- `develop/A`
- `develop/B`
- `develop/C`

git internally creates a `develop` folder and inside it puts the branches `A`, `B` and `C`. Since you have created a `develop` folder, there can no longer be a file (and therefore a branch) with the same name.

> You can test how you can't create a folder and a file with the same name in the same folder.

To solve this, we renamed the branch `develop/<something>` to `lab/<something>` and then we were able to create the `develop` branch without problems.

These are the commands used:

```bash
git switch develop/<something>
git branch -m lab/<something>
git checkout -b develop
```
